ui/js/graphs.js

```diff
@@ -468,16 +478,22 @@ perf.controller('GraphsCtrl', [
                              label: series.projectName + " " + series.name,
                              data: [],
                              resultSetData: [],
+                             customeasure:$scope.myMeasure,
```

Comment:
Actually, on reflection, we shouldn't need this variable per-series. We should just use $scope.myMeasure. So you can remove this line.